src/mosse.py

```diff
@@ -112,12 +108,27 @@ def detect(self, image):
 
         return self.region
 
+    def compute_response(self, image):
+        """
+            Update docstring
+        """
+        patch = crop_patch(image, self.region)
+
+        P = fft2(MOSSETrackerGrayscale.normalize(patch))
+        response = ifft2( np.conjugate(self.M) * P )
+
+        r, c = np.unravel_index(np.argmax(response), response.shape)
+
+        # Keep for visualisation
+        self.last_response = response
+        return self
```

Comment:
Should return `r, c` here instead?

Reply:
Hmm could be. I'll look into this! In that case I would have to make a minor refactoring of the code, but that's not a big deal :)